<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
    background-color:#ffffff;
}

QWidget#trading_acount {
    border-right: 2px solid #e9e9e9;
}



QWidget#list {
    border-left: 2px solid #e9e9e9;
}



QStackedWidget &gt; QWidget{
	background-color:#e9e9e9;
	border:none;
}

QWidget#trading_acount&gt;QWidget{
	border-bottom: 1px solid #e9e9e9;

}
QWidget#list&gt;QWidget{
	border-bottom: 1px solid #e9e9e9;
}
QWidget#list&gt;QWidget{
	
}

QPushButton{
    min-width: 100px;  /* Largeur minimale */
    min-height: 40px;  /* Hauteur minimale */
    max-width: 100px;  /* Largeur minimale */
    max-height: 40px;  /* Hauteur minimale */
border-radius:8px;
background-color:#f0f0f0;
border:none;
background-color:#f0f0f0;
}

QPushButton::hover{

background-color:#e0e0e0;
}

QPushButton#buy::hover{

background-color:#d7fcd7;
}
QPushButton#sell::hover{
background-color:#fcd7d7;
}
QPushButton#TP::hover{
background-color:#d7fcd7;
}
QPushButton#SL::hover{
background-color:#fcd7d7;
}
QPushButton#manuall::hover{
background-color:#fcd7d7;
}

QLineEdit{
    min-width: 200px;  /* Largeur minimale */
    min-height: 30px;  /* Hauteur minimale */
    max-width: 200px;  /* Largeur minimale */
    max-height: 30px;  /* Hauteur minimale */
background-color:#e1e1e1;
border-radius:8px;
}


</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="title">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="accounts_list">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectItems</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <item>
      <property name="text">
       <string>Nouvel élément</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nouvel élément</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nouvel élément</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nouvel élément</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nouvel élément</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nouvel élément</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nouvel élément</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nouvel élément</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nouvel élément</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nouvel élément</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nouvel élément</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nouvel élément</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nouvel élément</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="login_button">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="create_account_button">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
